nifi-nar-bundles/nifi-atlas-bundle/nifi-atlas-reporting-task/src/main/java/org/apache/nifi/atlas/reporting/ReportLineageToAtlas.java

```diff
@@ -585,12 +594,58 @@ private void initAtlasProperties(ConfigurationContext context) throws IOExceptio
         if (confDir != null) {
             // If atlasConfDir is not set, atlas-application.properties will be searched under classpath.
             Properties props = System.getProperties();
-            final String atlasConfProp = "atlas.conf";
+            final String atlasConfProp = ApplicationProperties.ATLAS_CONFIGURATION_DIRECTORY_PROPERTY;
             props.setProperty(atlasConfProp, confDir.getAbsolutePath());
             getLogger().debug("{} has been set to: {}", new Object[]{atlasConfProp, props.getProperty(atlasConfProp)});
         }
     }
 
+    private void parseAtlasUrls(final PropertyValue atlasUrlsProp, final Consumer<String> urlStrConsumer) {
```

Comment:
`parseAtlasUrls()` is now called with only a single type of consumers: `List::add`
It could be simplified by omitting the consumer parameter, creating the list instance here and returning it to the caller.